modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -512,7 +524,7 @@
 		"to": "fetchByExternalReferenceCode(param#1#, param#0#)"
 	},
 	{
-		"from": "AssetCategory::getLeftCategoryId",
+		"from": "regex:AssetCategory::getLeftCategoryId",
```

Comment:
previously it was being treated as a regex, im not sure if :: is a regex parameter or if it wanted to be treatable as a partial match because non-regexes are treated as hard matches but I basically just preserved the same functionality

Reply:
AFAIK the colon is not a special regex character. This seems like it's looking for a regular Method reference, which shouldn't need any special regex to find. I wonder if this is a workaround of some sort?